.travis.yml

```diff
@@ -2,3 +2,5 @@ language: node_js
 
 node_js:
   - '6'
+  - '4'
+  - '0.10'
```

Comment:
In terms of features, 0.10 and 4.0 are pretty similar. It's only syntax really, which was handled by transpiling with TypeScript anyway.